ooniapi/services/ooniprobe/src/ooniprobe/routers/v2.py

```diff
@@ -114,6 +122,7 @@ def get_vpn_config(
             "cert": provider.openvpn_cert,
             "key": provider.openvpn_key,
         },
-        endpoints=[format_endpoint(provider.provider_name, ep) for ep in provider.endpoints],
+        # Pick 4 random endpoints to serve to the client
```

Comment:
Picking 4 endpoints is great for MVP; perhaps we want to rethink the credential/endpoint turnover.
I think I would have `CREDENTIAL_FRESHNESS_INTERVAL_DAYS=1` so that we can exercise the mechanism more frequently.

How we conceptualize the choice is perhaps linked to the discussion in https://github.com/ooni/spec/pull/293#discussion_r1543574412 (i.e., a next step can be to select a bunch of IPs and measure all the port/transport combinations). The more I think about that discussion the more I'm inclined towards atomic ip-port-transport as the unit for a single measurement.